plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAlluxioCacheFileOperations.java

```diff
@@ -88,6 +75,22 @@ private void registerTable(String name, String resourcePath)
         getQueryRunner().execute(format("CALL system.register_table(CURRENT_SCHEMA, '%s', '%s')", name, dataPath));
     }
 
+    private void assertFileSystemAccesses(@Language("SQL") String query, Multiset<CacheOperation> expectedCacheAccesses)
+    {
+        assertUpdate("CALL system.flush_metadata_cache()");
+        DistributedQueryRunner queryRunner = getDistributedQueryRunner();
+        queryRunner.executeWithPlan(queryRunner.getDefaultSession(), query);
+        assertMultisetsEqual(getCacheOperations(), expectedCacheAccesses);
+    }
+
+    private Multiset<CacheOperation> getCacheOperations()
+    {
+        return getCacheOperationSpans(getQueryRunner())
+                .stream()
+                .map(DeltaLakeAlluxioCacheTestUtils::createCacheOperation)
+                .collect(toCollection(HashMultiset::create));
+    }
```

Comment:
Can we move this one to the util class ? We could define a method like `getCacheOperations(Queryrunner queryRunner)

Reply:
Done!